nab/detectors/htmcore/htmcore_detector.py

```diff
@@ -238,9 +251,19 @@ def modelRun(self, ts, val):
 
       # 3. Temporal Memory
       # Execute Temporal Memory algorithm over active mini-columns.
-      self.tm.compute(activeColumns, learn=True)
+      #self.tm.compute(activeColumns, learn=True)
```

Comment:
nit, maybe add if-else for panda_vis here? To explain the need for compute vs activate* , or is it just confusing?

Reply:
Good point 👍 , i realize that this code could potentially behave differently (not sure) so better to make the if-else. Thanks. The calculation order is difficult thing